api/globalConfig.py

```diff
@@ -146,14 +146,14 @@ def load_config(self):
         self.temp_dir = config.get('Directory-Section', 'temp', fallback='tests/resources/tmp')
         self.integrity_file_location = config.get('Web-Section', 'public-directory', fallback='tests/resources/html')
         self.diff_file_dir = config.get('Web-Section', 'save-diff-dir', fallback='tests/resources/html')
-        self.ip = config.get('Web-Section', 'ip', fallback='yangcatalog.org')
+        self.ip = config.get('Web-Section', 'ip', fallback='localhost')
         self.oidc_redirects = config.get('Web-Section', 'redirect-oidc', fallback='').split(' ')
         self.oidc_issuer = config.get('Web-Section', 'issuer', fallback='')
-        self.api_port = int(config.get('Web-Section', 'api-port', fallback=0))
+        self.api_port = int(config.get('Web-Section', 'api-port', fallback=8008))
```

Comment:
Set back to 0, this value is not used yet. It will be set in the future as needed.

Reply:
it is used in this file for creating suffix variable. Although it is not needed since we are using uwsgi (or gunicorn) we still need this to be set. 5000 is a default port on which every flask application is running.

Reply:
Value set to default 5000